score-vmware/src/main/java/io/cloudslang/content/vmware/entities/http/Protocol.java

```diff
@@ -0,0 +1,24 @@
+package io.cloudslang.content.vmware.entities.http;
+
+import io.cloudslang.content.vmware.constants.ErrorMessages;
+import org.apache.commons.lang3.StringUtils;
+
+/**
+ * Created by Mihai Tusa.
+ * 10/30/2015.
+ */
+public enum Protocol {
+    HTTP,
+    HTTPS;
+
+    public static String getValue(String input) throws Exception {
+        if (StringUtils.isBlank(input)) {
```

Comment:
is there a specific reason for throwing checked exceptions? if no, better use RuntimeException